.drone.yml

```diff
@@ -2,7 +2,7 @@ workspace:
   base: /go
   path: src/github.com/src-d/code-annotation
 
-branches: [master, staging, release/*]
+branches: [master, staging]
```

Comment:
why have you removed `release/*`?

Reply:
It was copied from Landing `.drone.yml`, that run tests, and triggers the deploy;
In Landing (and blog and talks), tests are run on `release/*` branches.
If we want to reproduce :point_up: this behavior, it should be done in `.travis.yml` file (because in CAT tests are run from Travis) but it is out of the scope of this PR (and we didn't consider it yet for this app)

Does it answer your question @rporres ?

Reply:
sure! :)